cmake/shared/PACE_CodeAnalysis.cmake

```diff
@@ -1,11 +1,60 @@
+#[=======================================================================[.rst:
+PACE_CodeAnalysis
+-----------------
+
+Run mlint and cppcheck code analysis on Project
+
+Variables required by the module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``${Herbert_ROOT}``
+This is provided by the `FindHerbert` module which must be loaded first
+
+``${PACE_MLINT_IGNORE}``
```

Comment:
what is this parameter? it's not immediately obvious what form the content should take or where (if anywhere) it's defined.